src/endpoints/sub_accounts.js

```diff
@@ -0,0 +1,59 @@
+const roote = "/subAccounts";
```

Comment:
I case what @emmqnuelkipsqng is referring @chepkok3 is 

name `/subaccount` instead of `subAccounts` as it is in the screenshot below 
![Uploading Screenshot 2022-11-20 at 10.07.45.png…]()